### PR TITLE
Add Brief Documentation Block to Test Property AWK Script

### PR DESCRIPTION
**Pull request by bska**
Mostly to aid future maintainers.

### CODE REVIEW

File: tests/getprop.awk [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+#
+#   search: Search pattern.  Typically a string such as
+#      set_tests_properties($test_name
+#
+#   prop: Property name, for instance DIRNAME or SIMULATOR.
+#
+# Property value will be printed to the standard output stream.
+#
+# The script assumes that $1 on candidate lines is suitable for matching
+# against 'search', and that $2 of the mathcing lines is the word
```
Comment (akva2) [inline tests/getprop.awk:14]: matching

**Review by akva2 — APPROVED** [review]
Except typo

File: tests/getprop.awk (hunk 1 above)
Comment (bska) [reply tests/getprop.awk]: > matching

Duh, thanks.  I've pushed an update to fix it.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**